packages/block-editor/src/components/inspector-controls-tabs/settings-tab.js

```diff
@@ -5,18 +5,26 @@ import AdvancedControls from './advanced-controls-panel';
 import PositionControls from './position-controls-panel';
 import { default as InspectorControls } from '../inspector-controls';
 import SettingsTabHint from './settings-tab-hint';
+import { BlockBindingsPanel } from '../block-bindings-panel';
 
-const SettingsTab = ( { showAdvancedControls = false } ) => (
-	<>
-		<InspectorControls.Slot />
-		<PositionControls />
-		{ showAdvancedControls && (
-			<div>
-				<AdvancedControls />
-			</div>
-		) }
-		<SettingsTabHint />
-	</>
-);
+function SettingsTab( { showAdvancedControls = false, block } ) {
```

Comment:
Should we be passing around `block` like this, or containing the lookup logic using a selector inside of the Block Bindings Panel component?